### DISCUSSION TITLE
jellyfin with softsub on browser

### DISCUSSION THREAD

**Discussion by JunkyardCat** [jellyfin/jellyfin#11350] [Q&A]
so is there a way for jellyfin to implement a feature which just like yt caption where you can highlight the sub?

**Comment by pypp** [comment]
What do you mean by highlight the sub ? you mean background for the subtitles ?

**Comment by JunkyardCat** [comment]
@pypp 
i mean i can select the text in the subtitle and copy its text.

**Reply by pypp** [reply]
To my knowledge there is no way of achieving this currently. YouTube uses custom captions while Jellyfin uses the built in cue system.
